app/layerfeaturesmodel.h

```diff
@@ -50,9 +50,16 @@ class LayerFeaturesModel : public FeaturesModel
 
   public:
 
+    enum LayerModelRoles
+    {
+      SearchResult = Qt::UserRole + 50, // pair of attribute and its value by which the feature was found, empty if search expression is empty
```

Comment:
I am afraid this might get complicated with different other derivated classes, could we use `FeaturesModel::LastRole + 1` instead of reusing the `UserRole` again?

Reply:
Changed in [5838b86](https://github.com/MerginMaps/mobile/pull/3800/commits/5838b864408c00a4404efb9050daff44c4d81f44)